src/apps/inventory/views.py

```diff
@@ -1,7 +1,16 @@
-from django.views.generic import DetailView
+from django.views.generic import DetailView, ListView
 
 from src.apps.inventory.models import Product
 
 
 class ProductDetailView(DetailView):
     model = Product
+
+
+class ProductBySellerListView(ListView):
+    model = Product
+    paginate_by = 10
+
+    def get_queryset(self):
+        if self.request.user.is_authenticated:
+            return Product.objects.filter(seller=self.request.user)
```

Comment:
return ...none() we still need it

Reply:
I dont think so, i have tested it, it works fine without this else statement